src/Microsoft.AspNetCore.Identity.UI/IdentityUserInterfaceConfigureRazorViewEngineOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Identity
+{
+    internal class IdentityUserInterfaceConfigureRazorViewEngineOptions : IConfigureOptions<RazorViewEngineOptions>
```

Comment:
.UI namespace?

Nit: 8 word name seems excessive, calling this ConfigureRazor is probably just as good?